src/Miew.js

```diff
@@ -3042,14 +3042,37 @@ Miew.prototype.screenshot = function (width, height) {
     return THREE.Math.radToDeg(Math.atan(tan)) * 2.0;
   }
 
+  function getScreenshotSafari() {
+    const canvas = document.createElement('canvas');
+    const canvasContext = canvas.getContext('2d');
+
+    canvas.width = width;
+    canvas.height = height;
+    canvasContext.drawImage(gfx.renderer.domElement, 0, 0, canvas.width, canvas.height);
+    const screenshot = canvas.toDataURL('image/png');
+
+    return screenshot;
+  }
   height = height || width || gfx.height;
   width = width || gfx.width;
 
+  const isSafariBrowser = navigator.vendor && navigator.vendor.indexOf('Apple') > -1
+    && navigator.userAgent
+    && navigator.userAgent.indexOf('CriOS') === -1
+    && navigator.userAgent.indexOf('FxiOS') === -1;
   let screenshotURI;
 
   if (width === gfx.width && height === gfx.height) {
-    // copy current canvas to screenshot
-    screenshotURI = gfx.renderer.domElement.toDataURL('image/png');
+    // renderer.domElement.toDataURL('image/png') returns flipped image in Safari
+    // It hasn't been resolved yet, but getScreenshotSafari()
+    // fixes it using an extra canvas.
+    // Also the bug can be fixed with premultipliedAlpha: true WebGL option
```

Comment:
Additionally, I ask to widen the comment with information about the fact: we use premultipliedAlpha: false because we need the option for rendering the image with transparent background.

Reply:
Do I need to widenthe comment or delete it, because such a bug fix would not work in this case?